.github/workflows/test-plugin.yml

```diff
@@ -50,7 +50,7 @@ jobs:
                 ln -s $(rustc --print sysroot)/lib/rustlib/src/rust/src $RUST_SRC_WITH_SYMLINK
                 ./gradlew :resolveDependencies -Pkotlin.incremental=false --no-daemon
                 ./gradlew assemble testClasses -Pkotlin.incremental=false --no-daemon
-                ./gradlew check -PexcludeTests=org/rustPerformanceTests/** -Pkotlin.incremental=false --no-daemon
+                ./gradlew check -PexcludeTests=org/rustPerformanceTests/** -PexcludeTests=org/rustSlowTests/cargo/runconfig/buildtool/CargoBuildManagerTest.class -PexcludeTests=org/rustSlowTests/ide/actions/RsBuildActionTest.class -Pkotlin.incremental=false --no-daemon
```

Comment:
I think we can do it much simpler and without this code duplication
Just check the existence of `CI` env variable in `CargoBuildTest`